### DISCUSSION TITLE
DateInput formatted view?

### DISCUSSION THREAD

**Discussion by brainlord** [vikramlearning/blazorbootstrap#379] [Q&A]
Greetings!

I was trying the library and I thought it is fantastic. I wanted to know since I did not find any reference in the documentation, if it is possible to customize the "dateinput" control so that only a month and a year can be selected, like the picture below...

![image](https://github.com/vikramlearning/blazorbootstrap/assets/28378223/3fb59165-49c4-40f3-b546-2448d42e17e3)

**Comment by gvreddy04** [comment]
@brainlord Right now, this feature is not supported. You can try something like this: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/month#try_it. 

The only issue here is browser compatibility.
**Reference:** https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/month#browser_compatibility
